### PR TITLE
readme: Introduce a release instruction document

### PR DESCRIPTION
**Pull request by EdDev**
Document the release process.

### CODE REVIEW

File: README.release.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+## Tagging
+
+- [ ] Tag new release in git.
+```bash
+# Make sure your local git repo is sync with upstream.
+# The whole version string should log like `v0.1.0`.
+# For the commit message use the following format: `kiagnose 0.1.0 release`.
+git tag --sign v<version>
```
Comment (orelmisan) [inline README.release.md:30]: Is there a typo here?

File: README.release.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- [ ] In case there is a need to remove a tag:
+```bash
+# Remove local tag
+git tag -d <tag_name>
+
+# Remove upstream tag
+git push --delete upstream <tag_name>
+```
+
+## GitHub Release
```
Comment (orelmisan) [inline README.release.md:43]: Is there a need for `https://quay.io/repository/kiagnose/kiagnose` section as well?

File: README.release.md (hunk 1 above)
Comment (EdDev) [reply README.release.md:30]: What typo?

File: README.release.md (hunk 2 above)
Comment (EdDev) [reply README.release.md:43]: It is suppose to be automatic AFAIK.
@qinqon ?

File: README.release.md (hunk 1 above)
Comment (orelmisan) [reply README.release.md:30]: Never mind, I understood now that the tag name is `v0.0.1` for example.
Sorry for the noise.

File: README.release.md (hunk 2 above)
Comment (EdDev) [reply README.release.md:43]: I will just add here a note to add ref to the images in the release, so users will know which one to use.
These images should be already available once the tag is pushed (if not, we need to fix it).

Comment (EdDev) [reply README.release.md:48]: Ok, added also a section.
Done.

Comment (orelmisan) [reply README.release.md:48]: Thank you for the change.
There are three places to look:
https://quay.io/repository/kiagnose/kiagnose?tab=tags
https://quay.io/repository/kiagnose/echo-checkup?tab=tags
https://quay.io/repository/kiagnose/kubevirt-vm-latency?tab=tags

Comment (EdDev) [reply README.release.md:50]: Done

**Review by orelmisan — APPROVED** [review]
Thank you!

**Pull request merged**